docs/source/tutorial_model_cards.md

```diff
@@ -0,0 +1,181 @@
+# Generating Model Cards Semi-Automatically
+
+**Author**: Wendy Zhang
+
+Here's a sample model card for the [Blenderbot2.0 2.7B model](https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/blenderbot2/model_card.md).
```

Comment:
hmm is it worth first giving a sentence or two explaining what model cards are?

Reply:
true... true :)